substrafl/nodes/test_data_node.py

```diff
@@ -31,11 +31,15 @@ def __init__(
         organization_id: str,
         data_manager_key: str,
         test_data_sample_keys: List[str],
-        metric_keys: List[str],  # key to the metric, use substra.Client().add_algo()
+        metric_keys: List[str],
     ):
         self.data_manager_key = data_manager_key
         self.test_data_sample_keys = test_data_sample_keys
-        self.metric_keys = metric_keys
+
+        if isinstance(metric_keys, list):
+            self.metric_keys = metric_keys
+        else:
+            raise TypeError("metric keys must be of type list")
```

Comment:
```suggestion
        if not isinstance(metric_keys, list):
            raise TypeError("metric keys must be of type list")
        self.metric_keys = metric_keys
```

Very minor but helping readability and reducing "nestedness" a bit (from _The Zen of Python_ : "Flat is better than nested")

Reply:
Nice thanks ! 😄